packages/@react-spectrum/searchwithin/stories/SearchWithin.stories.tsx

```diff
@@ -0,0 +1,78 @@
+ */
+import {action} from '@storybook/addon-actions';
+import {Item, Picker} from '@react-spectrum/picker';
+import React from 'react';
+import {SearchField} from '@react-spectrum/searchfield';
+import {SearchFieldProps} from '@react-types/searchfield';
+import {SearchWithin} from '../';
+import {SpectrumPickerProps} from '@react-types/select';
+import {SpectrumSearchWithinProps} from '@react-types/searchwithin';
+
+export default {
+  title: 'SearchWithin'
+};
+
+function render(props: Omit<SpectrumSearchWithinProps, 'children'> = {}, searchFieldProps: SearchFieldProps = {}, pickerProps: Omit<SpectrumPickerProps<object>, 'children'> = {}) {
+  return (
+    <SearchWithin label="Search" {...props}>
+      <SearchField placeholder="Search" {...searchFieldProps} onChange={action('change')} onSubmit={action('submit')} />
+      <Picker defaultSelectedKey="all" {...pickerProps} onSelectionChange={action('selectionChange')}>
+        <Item key="all">All</Item>
+        <Item key="campaigns">Campaigns</Item>
+        <Item key="audiences">Audiences</Item>
+        <Item key="tags">Tags</Item>
+      </Picker>
+    </SearchWithin>
+  );
+}
+
+function renderReverse(props: Omit<SpectrumSearchWithinProps, 'children'> = {}, searchFieldProps: SearchFieldProps = {}, pickerProps: Omit<SpectrumPickerProps<object>, 'children'> = {}) {
+  return (
+    <SearchWithin label="Search" {...props}>
+      <Picker defaultSelectedKey="all" {...pickerProps} onSelectionChange={action('selectionChange')}>
+        <Item key="all">All</Item>
+        <Item key="campaigns">Campaigns</Item>
+        <Item key="audiences">Audiences</Item>
+        <Item key="tags">Tags</Item>
+      </Picker>
+      <SearchField placeholder="Search" {...searchFieldProps} {...searchFieldProps} onChange={action('change')} onSubmit={action('submit')} />
+    </SearchWithin>
+  );
+}
+
+export const Default = () => render({});
+
+export const isDisabled = () => render({isDisabled: true});
+isDisabled.storyName = 'isDisabled: true';
+
+export const isRequired = () => render({isRequired: true});
+isRequired.storyName = 'isRequired: true';
+
+export const isRequiredNecessityIndicatorLabel = () => render({isRequired: true, necessityIndicator: 'label'});
+isRequiredNecessityIndicatorLabel.storyName = 'isRequired: true, necessityIndicator \'label\' ';
+
+export const isRequiredFalse_necessityIndicator = () => render({isRequired: false, necessityIndicator: 'label'});
+isRequiredFalse_necessityIndicator.storyName = 'isRequired: false, necessityIndicator \'label\' ';
+
+export const PickerDisabled = () => render({}, {}, {isDisabled: true});
+
+export const CustomWidth = () => render({width: 300});
+
+export const LabelPositionSide = () => render({labelPosition: 'side'});
+LabelPositionSide.storyName = 'labelPosition: side';
+
+export const NoLabel = () => render({label: undefined, 'aria-label': 'Aria Label'});
+
+export const AutoFocus = () => render({}, {autoFocus: true});
```

Comment:
You can't autofocus the picker, only the input?

Reply:
Good point, I'll add a story for that and rename this.